modules/gdnavigation/nav_map.cpp

```diff
@@ -657,6 +657,7 @@ void NavMap::sync() {
 					connection->get().B->edges[connection->get().B_edge].other_edge = connection->get().A_edge;
 				} else {
 					// The edge is already connected with another edge, skip.
+					ERR_PRINT("I'm trying to merge a navigation mesh triangle edge with another one already merged. This happens when the `Navigation` `cell_size` is different from the one used to generate the navigation mesh. This will cause navigation problem.");
```

Comment:
I wouldn't make the engine speak at the first person... we don't do this anywhere and this doesn't sound professional.

```suggestion
					ERR_PRINT("Attempted to merge a navigation mesh triangle edge with another already-merged edge. This happens when the Navigation's `cell_size` is different from the one used to generate the navigation mesh. This will cause navigation problem.");
```

Reply:
Yes, I agree.. I didn't know how to write it.